auth-web/src/components/auth/common/HelpDialog.vue

```diff
@@ -1,25 +1,29 @@
 <template>
-  <v-dialog v-model="isDialogOpen" max-width="640">
+  <v-dialog v-model="isDialogOpen" max-width="45rem">
```

Comment:
This makes it a bit wider than the underlying parent dialog, which looks better.